spec/lib/lightweight_serializer/serializer_spec.rb

```diff
@@ -413,6 +413,30 @@ def translate(message)
       end
     end
 
+    context 'when called as_json with a DateTime object' do
+      let(:my_date) { DateTime.now }
+      let(:date_object) { OpenStruct.new(my_date: my_date) }
```

Comment:
It's a nitpick but date_object sounds like a `DateTime` instance, I'd call it `object_with_date`